relational_operators/CMakeLists.txt

```diff
@@ -263,6 +263,8 @@ target_link_libraries(quickstep_relationaloperators_SelectOperator
                       glog
                       quickstep_catalog_CatalogRelation
                       quickstep_catalog_CatalogTypedefs
+                      quickstep_catalog_NUMAPlacementScheme
```

Comment:
@adalbertgerald Please add the macro `QUICKSTEP_HAVE_LIBNUMA` around `quickstep_catalog_NUMAPlacementScheme`.